org.lflang/build.gradle

```diff
@@ -81,3 +81,13 @@ task generateStandaloneCompiler() {
 }
 
 generateStandaloneCompiler.finalizedBy shadowJar
+
+task runLfc(type: JavaExec) {
```

Comment:
Interesting. Do you know that the `lfc` script is capable of something similar? If you run `lfc -r`, it rebuilds itself via gradle before running.

Reply:
Ah ok I didn't know. I added the gradle task to be able to run & debug it from intellij, I'm not sure it would be possible with a bash script